.github/workflows/autobuild.yaml

```diff
@@ -122,26 +122,36 @@ jobs:
         "-DCMAKE_CXX_FLAGS_RELEASE=${{ matrix.config.cflags }}"
         ${{ matrix.config.cmake }}
     - name: Ninja
-      run: ninja -C ${{ env.CLANGD_DIR }} clangd
-    - name: Archive
+      run: ninja -C ${{ env.CLANGD_DIR }} clangd clangd-indexer
+    - name: Archive clangd
       run: >
         7z a clangd.zip
         ${{ env.CLANGD_DIR }}/LICENSE.TXT
-        ${{ env.CLANGD_DIR }}/bin/clangd*
+        ${{ env.CLANGD_DIR }}/bin/clangd
+        ${{ env.CLANGD_DIR }}/bin/clangd.exe
```

Comment:
i don't think this will work as `7z` will produce a non-zero exit code on non-existent files and fail the workflow (same for clangd-indexer.exe below)

i would define a `binary_extension` on jobs matrix and use that instead.